ansible/server-state-playbooks/ome-dundeeomero/playbook.yml

```diff
@@ -135,7 +135,8 @@
     # Note - had to have these set to `install-mock` to progress role
     # installation before changing config to restored DB from other system.
     - role: openmicroscopy.omero-server
-      omero_server_release: 5.3.1
+      omero_server_release: 5.3.2
+      omero_server_upgrade: True
```

Comment:
Sorry, I missed this comment before say ":thumbsup:". I may be misremembering but I would consider this flag dangerous on a production system. I suggest reverting it.